_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+
+# Projects
+
+Tons of open source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large open source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These were great because students and mentors could work together to contribute to some of the tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing open source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn). We love these open source projects because they get great opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to just contributing to open source, though. Many students developed their own open source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul) even has 69 stars on Github and was mentioned in a talk at GoogleIO! And it was born right here at HackIllinois.
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/mentors-open-source-hackillinois/puppeteer.jpg "puppeteer")
+
+# Sponsors
+
+Sponsors at HackIllinois had a great presence in Open Source as well! Companies had tons of opportunities to get involved with hackers on their projects. Nearly all companies brought mentors that worked with students on contributing and creating open source projects. Even companies that didn't have their own open source projects were able to work with students, because the mentors that came had experience working on large scale projects and were able to contribute alongside students. The one-on-one attention and guidance that these mentors offered was integral to many of the projects that won prizes at HackIllinois.
+
+Companies also had the option to interact with hackers in many other ways too. Some companies sponsored a prize and were able to present it to the project of their choice at the end of the event, and some companies even had personalized lounges for hackers to work in!
+
+<center>&middot;&middot;&middot;</center>
+
+We're incredibly happy that open source was such a success at HackIllinois 2018, and aim to continue the contributions next year at HackIllinois 2019! In the meantime, make sure to check out [our other recent post](https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html) by staff member Justin Yang recapping the entirety of HackIllinois 2018.
```

Comment:
This paragraph would be a good place to encourage readers to continue or start contributing before HackIllinois 2019.

> https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html

I believe there's a way to reference this post relatively.